crates/ruma-client-api/src/sync/sync_events/v4.rs

```diff
@@ -468,6 +468,15 @@ pub struct SlidingSyncRoom {
     /// The number of timeline events which have just occurred and are not historical.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub num_live: Option<UInt>,
+
+    /// The timestamp of the room.
+    ///
+    /// It's not to be confused with `origin_server_ts` of the latest event in the
+    /// timeline. `bump_event_types` might "ignore” some events when computing the
+    /// timestamp of the room. Thus, using this `timestamp` value is more accurate than
+    /// relying on the latest event.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub timestamp: Option<UInt>,
```

Comment:
This is probably a number in milliseconds since unix epoch, like `origin_server_ts`? We have a type for that, `MillisecondsSinceUnixEpoch`.

Reply:
Fixed up. Thanks.